nglview/show.py

```diff
@@ -5,6 +5,10 @@
 except ImportError:
     from io import StringIO
 
+from StringIO import StringIO as StringIOOld
+
+from Bio.PDB import PDBIO
```

Comment:
Move this into `show_biopython` so `biopython` is just optional package in nglview.